ansible/roles/test/files/saitests/pfc_asym.py

```diff
@@ -160,7 +157,9 @@ def runTest(self):
 
         # Verify that PFC frames are not generated for lossy priorities
         for sp in self.server_ports:
-            port_counters, queue_counters = sai_thrift_read_port_counters(self.client, port_list[int(sp['index'])])
+            port_counters, queue_counters = sai_thrift_read_port_counters(self.client, sp['oid'])
+            self.log("\Lossy: port_counters = {}\n queue_counters = {}\nserver_ports = {}".format(port_counters,
```

Comment:
It will be hard to read, line will be to long.
It is good readable even when there are many ports, because it goes one by one.

Also fixed formatting here.